cg/meta/compress/compress.py

```diff
@@ -81,8 +81,11 @@ def compress_fastq(self, sample_id: str) -> bool:
                 )
                 all_ok = False
                 continue
-
-            if not self.crunchy_api.is_fastq_compression_possible(compression_obj=compression):
+            is_spring_archived: bool = self._is_spring_archived(compression)
+            is_compression_possible: bool = self.crunchy_api.is_fastq_compression_possible(
+                compression_obj=compression, is_spring_archived=is_spring_archived
+            )
+            if not is_compression_possible:
```

Comment:
I suggest a small re-write of this function.
1. Remove the empty-file check above
2. Create a function here that checks if the file is compressible (ie `encompasses` is_spring_archived and crunchy_api.`is_fastq_compression_possible`)
3. Change almost all of the log levels to debug

Reply:
@Vince-janv Could you check now if I understood correctly?

Reply:
And also - why should we skip the empty-file check?

Reply:
Fair enough! I will remove it

Reply:
Removed it! Passed the sample_id for better logging though...